modules/i3s/src/i3s-loader.ts

```diff
@@ -31,12 +31,11 @@ export const I3SLoader: LoaderWithParser = {
   extensions: ['bin'],
   options: {
     i3s: {
-      loadContent: true,
       token: null,
       isTileset: 'auto',
       isTileHeader: 'auto',
-      tile: null,
-      tileset: null,
+      tileOptions: null,
```

Comment:
Are these options documented? If so it is a breaking change, maybe we need to still support the old ones and mark them as deprecated.

If not documented, should we document the new options?

If they are internal and not for users, prefix with underscore?